core/client/app/controllers/post-settings-menu.js

```diff
@@ -290,51 +291,59 @@ export default Controller.extend(SettingsMenuMixin, {
          * (#1351)
          */
         setPublishedAt(userInput) {
-            let newPublishedAt = parseDateString(userInput);
-            let publishedAt = moment(this.get('model.publishedAt'));
-            let errMessage = '';
-
-            if (!userInput) {
-                // Clear out the publishedAt field for a draft
-                if (this.get('model.isDraft')) {
-                    this.set('model.publishedAt', null);
+            let offset = RSVP.resolve(this.get('timeZone.offset'));
```

Comment:
The `RSVP.resolve` isn't necessary here as `timeZone.offset` always returns a promise. You could remove this line and simplify the following line to:

```
this.get('timeZone.offset').then((offset) => {
```